.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
```

Comment:
Will these be "click on" etc, given it's a bug with a Terraform provider?

Reply:
@mcncl what about this?

```suggestion
Please provide a minimum `.tf` file to reproduce the issue.  
Provide any other instructions necessary, such as what fields/values to change to trigger the bug
```

Reply:
@jradtilbrook I think https://github.com/hashicorp/terraform-provider-aws/issues/31537 is a good reference for what to include.

That's the official AWS provider, so well used. Also a reasonable standard to hold ourselves to